gobblin-api/src/main/java/gobblin/commit/CommitSequenceStore.java

```diff
@@ -0,0 +1,65 @@
+/*
+ * Copyright (C) 2014-2015 LinkedIn Corp. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use
+ * this file except in compliance with the License. You may obtain a copy of the
+ * License at  http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed
+ * under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR
+ * CONDITIONS OF ANY KIND, either express or implied.
+ */
+
+package gobblin.commit;
+
+import java.io.IOException;
+import java.util.Collection;
+
+import com.google.common.base.Optional;
+
+
+/**
+ * A store for {@link CommitSequence}s. A {@link CommitSequence} is identified by job name and dataset URN.
+ *
+ * @author ziliu
+ */
+public interface CommitSequenceStore {
+
+  /**
+   * Whether a job name exists in the store.
```

Comment:
Can you clarify what it means for a job name to exist in the store? There is a CommitSequence for some dataset?